diff --git a/node_modules/drizzle-kit/bin.cjs b/node_modules/drizzle-kit/bin.cjs
index 07274ea..535334a 100644
--- a/node_modules/drizzle-kit/bin.cjs
+++ b/node_modules/drizzle-kit/bin.cjs
@@ -17858,6 +17858,9 @@ WHERE
             for (const checks of tableChecks) {
               let checkValue = checks.constraint_definition;
               const constraintName = checks.constraint_name;
+              if (checkValue == null || typeof checkValue !== "string") {
+                continue;
+              }
               checkValue = checkValue.replace(/^CHECK\s*\(\(/, "").replace(/\)\)\s*$/, "");
               checkConstraints[constraintName] = {
                 name: constraintName,
